packages/devkit/src/utils/config-utils.ts

```diff
@@ -54,15 +54,23 @@ export function getRootTsConfigFileName(): string | null {
   return null;
 }
 
+const packageInstallationDirectories = ['node_modules', '.yarn'];
+
+export function clearRequireCache() {
+  for (const k of Object.keys(require.cache)) {
+    if (!packageInstallationDirectories.some((dir) => k.includes(dir))) {
```

Comment:
Please add a todo comment here which says that we're filtering it for `jest` and that it should be removed after plugin isolation is everywhere.

Reply:
After talking w/ Jack, I think it makes more sense to keep this here until at least plugin iso is enabled, but there are other benefits (speed) to keeping it around so maybe we just leave it?